.github/workflows/run_protoc_java-monitoring.yaml

```diff
@@ -0,0 +1,14 @@
+on:
+  pull_request:
+jobs:
+  run-protoc:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Run run_protoc.sh
+      shell: bash
+      run: |
+        cd java-monitoring
+        sh -x run_protoc.sh
+        echo "Code generation finished"
```

Comment:
@JoeWang1127 I've added this check to help you make it runnable. It's failing here: 

```
...
+ git clone git@github.com:googleapis/googleapis.git
Cloning into 'googleapis'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

https://github.com/googleapis/google-cloud-java/actions/runs/5029970368/jobs/9022071254